ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -190,6 +189,73 @@
         <A.Body @text={{t 'resources.target.messages.none.description'}} />
       </Hds::ApplicationState>
     {{/if}}
+    {{#if (and this.selectedTarget this.selectedTarget.isActive)}}
```

Comment:
Could this just check just be 
```suggestion
    {{#if this.selectedTarget}}
```
We can't select targets that aren't active right? You've already filtered them ones that you can select?

Reply:
There could be a scenario where a user has the flyout open, and cancels the only session in the table and that check is there to close the flyout.

Reply:
Gotcha makes sense!